components/SortMenu.tsx

```diff
@@ -65,21 +34,21 @@ export const SortMenu = <T,>({
       >
         Sort by <ChevronDownIcon />{' '}
       </MenuButton>
-      <MenuList w="fit-content">
+      <MenuList zIndex={2}>
         <MenuOptionGroup
           title="Order"
           type="radio"
           defaultValue="asc"
-          onChange={(e) => setOrder(e)}
+          onChange={(e) => setOrder(e as SortOrder)}
         >
           <MenuItemOption value="asc">Ascending</MenuItemOption>
           <MenuItemOption value="desc">Descending</MenuItemOption>
         </MenuOptionGroup>
         <MenuDivider />
         <MenuOptionGroup
           title="Sort by"
-          type="checkbox"
-          onChange={(e) => setSortFields(e)}
+          type="radio"
+          onChange={(e) => setSortField(e)}
```

Comment:
Changing this from a `checkbox` to to `radio` type because it doesn't make sense to allow sorting on multiple fields (name and description) of the same type.